app/assets/stylesheets/launcher.scss

```diff
@@ -412,7 +412,7 @@
       position: absolute;
       top: -3px;
       left: -20px;
-      -webkit-transform: rotate(90deg);
+      // -webkit-transform: rotate(90deg);
```

Comment:
Please don't leave code commented out in a Pull Request.  It should either be removed or be uncommented.